src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -458,6 +458,14 @@ protected function instantiateObject(array &$data, string $class, array &$contex
 
         unset($context['has_constructor']);
 
+        if (!$reflectionClass->isInstantiable()) {
+            throw NotNormalizableValueException::createForUnexpectedDataType(
+                sprintf('Failed to create object because the class "%s" is not instantiable.', $class),
+                $data,
+                []
```

Comment:
```suggestion
                ['unknown'],
                $context['deserialization_path'] ?? null,
                false,
                $e->getCode(),
                $e,
```

Reply:
I've added `['unknown'],` and `$context['deserialization_path'] ?? null,`
There is no exception though so can't add the other two